cont3xt/vueapp/src/utils/InputDropdown.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <b-dropdown>
+    <b-input-group size="sm">
+      <b-input-group-text>
+        <span class="fa fa-search" />
+      </b-input-group-text>
+      <b-form-input
+          ref="integrationViewDropdownSearch"
+          @keydown.enter="selectTopOption"
+          @focusin="setBarFocused"
+          @focusout="setBarUnfocused"
+          v-model="filter"
+      />
+    </b-input-group>
+
+<!--    <b-dropdown-item v-for="option in options">-->
```

Comment:
what's this for?

Reply:
WIP from auto-complete for overview fields, I've added a todo